tests/python/pants_test/reporting/test_reporting_integration.py

```diff
@@ -154,3 +163,46 @@ def test_epilog_to_stderr(self, quiet_flag):
     self.assert_success(pants_run)
     self.assertIn('Cumulative Timings', pants_run.stderr_data)
     self.assertNotIn('Cumulative Timings', pants_run.stdout_data)
+
+  def test_zipkin_reporter(self):
+    with http_server(ZipkinHandler) as port:
+      endpoint = "http://localhost:{}".format(port)
+      command = [
+        '--reporting-zipkin-endpoint={}'.format(endpoint),
+        '--reporting-zipkin-encoding=json',
+        'cloc',
+        'src/python/pants:version'
+      ]
+
+      pants_run = self.run_pants(command)
+      self.assert_success(pants_run)
+
+      num_of_traces = len(ZipkinHandler._traces)
+      self.assertEqual(num_of_traces, 1)
+
+      trace = ZipkinHandler._traces[-1]
+      main_span = self.find_span_by_name(trace, 'main')
+      self.assertEqual(len(main_span), 1)
+
+      parent_id = main_span[0]
+      main_children = self.find_span_by_parentId(trace, parent_id)
+      self.assertTrue(main_children)
+      self.assertTrue(any(name == 'cloc' for name in main_children))
+
+  @staticmethod
+  def find_span_by_name(trace, name):
+    return [span['id'] for span in trace if span['name'] == name]
+
+  @staticmethod
+  def find_span_by_parentId(trace, parent_id):
+    return [span['name'] for span in trace if span.get('parentId') == parent_id]
+
+
+class ZipkinHandler(BaseHTTPRequestHandler):
+  _traces = []
```

Comment:
Let's call this `traces`, not `_traces`, because it's part of the public interface of the type